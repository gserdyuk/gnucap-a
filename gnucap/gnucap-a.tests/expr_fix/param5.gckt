* param with ? expression - wrong () handling


*.param distr(b, n, w, skew1bm1w)  = 'n + skew1bm1w*2.*(w-n)*(b-n)/(w==n||n==b||w==b?1.:w-b+skew1bm1w*(w+b-2.*n))'
.param distr213  = '1 + 0.1*2.*(3-1)*(2-1)/(3==1||1==2||3==2?1.:3-2+0.1*(3+2-2.*1))'

v1 1 0 dc 'distr213'

.print op v(v*) 

.op

.end
