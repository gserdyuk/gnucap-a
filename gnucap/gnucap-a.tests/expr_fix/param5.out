Gnucap gnucap-arails  2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* param with ? expression - wrong () handling 
... 0.1*2.*(3-1)*(2-1)/(3==1||1==2||3==2?1.:3-2+0.1*(3+2-2.*
                                        ^ ? unbalanced parentheses (factor)
