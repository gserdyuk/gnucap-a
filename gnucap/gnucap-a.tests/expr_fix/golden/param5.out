Gnucap gnucap-arails  2009.12.07 RCS 26.136 globalpar
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* param with ? expression - wrong () handling 
#           v(v1)      v(v2)     
 27.        1.3077     1.3077    
